app/helpers/kadmin/bootstrap_helper.rb

```diff
@@ -34,5 +34,29 @@ def dropdown(prompt, links = [])
 
       return content_tag(:div, button + list, class: 'dropdown', style: 'display: inline-block;')
     end
+
+    def help_assitance(label, required, title, message)
+      # binding.pry
+      label = t(title)
+      label = label.html_safe
+      
+      message = t(message)
+      message = message.gsub('"', '\"')
+      message = message.gsub("'", "\'")
+      html_message = message.html_safe
+
+      title = t(title)
+      title = title.html_safe
+    
+      require_html = required ? '<span class="required-field"><span>':''      
+      icon_html = "<i class='fa fa-question-circle' style='color:green'></i>"
+      
+      display_message = "<span data-toggle='popover' title='#{title}' data-placement='top'
+      data-trigger='hover' data-content='#{html_message}'>
+       #{icon_html}
+      </span>"
+
+      display_message = "#{require_html}#{label} #{display_message}".html_safe
```

Comment:
If this is the return value of your method, it's clearer to just use `return`.  Above would work, but it's confusing to read imo.